packages/packages/top-bar/src/__tests__/index.test.tsx

```diff
@@ -2,15 +2,14 @@ import { useState } from 'react';
 import {
```

Comment:
Create tests for the `createMenu` API
and separate tests for the components that render the menus if needed

Reply:
Done